alicloud/resource_alicloud_adb_account.go

```diff
@@ -0,0 +1,219 @@
+package alicloud
+
+import (
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/aliyun/alibaba-cloud-sdk-go/services/adb"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/validation"
+	"github.com/terraform-providers/terraform-provider-alicloud/alicloud/connectivity"
+)
+
+func resourceAlicloudAdbAccount() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceAlicloudAdbAccountCreate,
+		Read:   resourceAlicloudAdbAccountRead,
+		Update: resourceAlicloudAdbAccountUpdate,
+		Delete: resourceAlicloudAdbAccountDelete,
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"db_cluster_id": {
+				Type:     schema.TypeString,
+				ForceNew: true,
+				Required: true,
+			},
+
+			"account_name": {
+				Type:     schema.TypeString,
+				ForceNew: true,
+				Required: true,
+			},
+
+			"account_password": {
+				Type:      schema.TypeString,
+				Required:  true,
+				Sensitive: true,
```

Comment:
The password should also support kms_password, like https://github.com/terraform-providers/terraform-provider-alicloud/blob/master/alicloud/resource_alicloud_db_account.go#L44